src/mergeProp.js

```diff
@@ -0,0 +1,28 @@
+import { curry } from 'ramda';
+import mergePath from './mergePath';
+
+/**
+ * Merges user provided data (subject) with some object (target)
```

Comment:
Create a new object with the own properties of the object under the `p` merged with the own properties of the provided `source`. If a key exists in both objects, the value from the `source` object will be used.